projects/public-search/src/app/document-brief/document-brief.component.html

```diff
@@ -15,63 +15,52 @@
+      <li *ngFor="let contribution of record.metadata.contribution.slice(0,3); let last = last; let i = index"
+          class="list-inline-item">
+        <ng-container *ngIf="!contribution.agent.pid; else contribLink">
+          {{ contribution | contributionFormat }}
+        </ng-container>
+        <ng-template #contribLink>
+          <a *ngIf="contribution.agent.pid"
+             href="/{{ viewcode }}/{{ contribution.agent.type | contributionType }}/{{ contribution.agent.pid }}">
             {{ contribution | contributionFormat }}
           </a>
-          </ng-template>
-          {{ last ? '' : '; ' }}
-          {{ last && record.metadata.contribution.length > 3 ? '; …' : '' }}
-        </li>
-      </ul>
-
-      <!-- provision activity publications -->
-      <ul class="list-unstyled" *ngIf="provisionActivityPublications.length > 0">
-        <li *ngFor="let publication of provisionActivityPublications">{{ publication.value }}</li>
-      </ul>
-
-      <!-- explanation -->
-      <div *ngIf="record.explanation">
-        <a class="badge badge-info collapsed"
-            data-toggle="collapse" href="#{{'score'+record.metadata.pid }}"
-            aria-expanded="false">
-            score: {{ record.explanation.value }}
-        </a>
-        <pre class="collapse border border-secondary mt-1" id="{{'score'+record.metadata.pid }}">{{record.explanation|json}}</pre>
-      </div>
-
-      <!-- part of -->
-      <shared-part-of [record]="record" [viewcode]="viewcode" [isPublicView]="true"></shared-part-of>
-
-      <!-- availability -->
-      <div *ngIf="record.metadata.type !== 'ebook'">
-        <i class="fa fa-circle text-{{ record.metadata.available ? 'success' : 'danger' }}" aria-hidden="true"></i>&nbsp;
-        <span translate *ngIf="record.metadata.available">available</span>
-        <span translate *ngIf="!record.metadata.available">not available</span>
-      </div>
-    </article>
+        </ng-template>
+        {{ last ? '' : '; ' }}
+        {{ last && record.metadata.contribution.length > 3 ? '; …' : '' }}
+      </li>
+    </ul>
+    <!-- provision activity publications -->
+    <ul class="list-unstyled" *ngIf="provisionActivityPublications.length > 0">
+      <li *ngFor="let publication of provisionActivityPublications">{{ publication.value }}</li>
+    </ul>
+    <!-- explanation -->
+    <div *ngIf="record.explanation">
+      <a class="badge badge-info collapsed"
+         data-toggle="collapse" href="#{{ 'score' + record.metadata.pid }}"
+         aria-expanded="false">
+         score: {{ record.explanation.value }}
+      </a>
+      <pre class="collapse border border-secondary mt-1" id="{{ 'score' + record.metadata.pid }}">
+        {{ record.explanation | json }}
+      </pre>
+    </div>
+    <!-- part of -->
+    <shared-part-of [record]="record" [viewcode]="viewcode" [isPublicView]="true"></shared-part-of>
+    <!-- availability -->
+    <div *ngIf="record.metadata.type !== 'ebook'">
```

Comment:
Are you sure that `ebook` is the right type (see 'document types' json schema)?

Reply:
I just use the same code than original file. Don't change this `ngIf` statement